pkgs/build-support/trivial-builders.nix

```diff
@@ -121,7 +121,7 @@ rec {
         allowSubstitutes = false;
       }
       ''
-        target=$out${destination}
+        target="$out${destination}"
```

Comment:
```suggestion
        target=$out${lib.escapeShellArg destination}
```

Reply:
(let's not break again when destination contains a `\` or a `"`)

Reply:
Oh no, I just realized this is a Nix interpolation.

Reply:
Thanks, applied.

Reply:
If it was just bash, there'd be no need for quotes either: interpolations in variable assignments are not subject to further expansion. (The rule of thumb is that you should `"$quote" "${every}" "$((bloody))" "$(expansion)"` wherever bash expects multiple strings (commands, arrays, ...), but variable assignments expect a single string)